<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="textBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(236, 239, 241);
font: 300 24pt &quot;Segoe UI&quot;;
background-color: rgb(33, 33, 33);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sqrt_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
background-color: rgb(245, 124, 0);
color: rgb(255, 255, 255);
border-radius: 25px;
}

</string>
   </property>
   <property name="text">
    <string>√</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
background-color: rgb(245, 124, 0);
color: rgb(255, 255, 255);
border-radius: 25px;
}

</string>
   </property>
   <property name="text">
    <string>log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cos_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
background-color: rgb(245, 124, 0);
color: rgb(255, 255, 255);
border-radius: 25px;
}

</string>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cot_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
background-color: rgb(245, 124, 0);
color: rgb(255, 255, 255);
border-radius: 25px;
}

</string>
   </property>
   <property name="text">
    <string>cot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sin_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
background-color: rgb(245, 124, 0);
color: rgb(255, 255, 255);
border-radius: 25px;
}

</string>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tan_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
background-color: rgb(245, 124, 0);
color: rgb(255, 255, 255);
border-radius: 25px;
}

</string>
   </property>
   <property name="text">
    <string>tan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ac_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
background-color: rgb(69, 90, 100);
color: rgb(255, 255, 255);
border-radius: 25px;
}

</string>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="neg_pos_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
background-color: rgb(245, 124, 0);
color: rgb(255, 255, 255);
border-width: 16px;
border-radius: 25px;
}

</string>
   </property>
   <property name="text">
    <string>±</string>
   </property>
  </widget>
  <widget class="QPushButton" name="percentage_btn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
background-color: rgb(245, 124, 0);
color: rgb(255, 255, 255);
border-radius: 25px;
}

</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="division_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
background-color: rgb(245, 124, 0);
color: rgb(255, 255, 255);
border-radius: 25px;
}

</string>
   </property>
   <property name="text">
    <string>÷</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cross_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
background-color: rgb(245, 124, 0);
color: rgb(255, 255, 255);
border-radius: 25px;
}

</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sub_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
background-color: rgb(245, 124, 0);
color: rgb(255, 255, 255);
border-radius: 25px;
}

</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sum_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
background-color: rgb(245, 124, 0);
color: rgb(255, 255, 255);
border-radius: 25px;
}

</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equal_btn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(38, 50, 56);
border-radius: 25px;
}
</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(38, 50, 56);
border-radius: 25px;
}
</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(38, 50, 56);
border-radius: 25px;
}
</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(38, 50, 56);
border-radius: 25px;
}
</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(38, 50, 56);
border-radius: 25px;
}
</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(38, 50, 56);
border-radius: 25px;
}
</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(38, 50, 56);
border-radius: 25px;
}
</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(38, 50, 56);
border-radius: 25px;
}
</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(38, 50, 56);
border-radius: 25px;
}
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(38, 50, 56);
border-radius: 25px;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dot_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(38, 50, 56);
border-radius: 25px;
}
</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 300 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(38, 50, 56);
border-radius: 25px;
}
</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-20</y>
     <width>371</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(84, 110, 122);


</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>textBox</zorder>
  <zorder>sqrt_btn</zorder>
  <zorder>log_btn</zorder>
  <zorder>cos_btn</zorder>
  <zorder>cot_btn</zorder>
  <zorder>sin_btn</zorder>
  <zorder>tan_btn</zorder>
  <zorder>ac_btn</zorder>
  <zorder>neg_pos_btn</zorder>
  <zorder>percentage_btn</zorder>
  <zorder>division_btn</zorder>
  <zorder>cross_btn</zorder>
  <zorder>sub_btn</zorder>
  <zorder>sum_btn</zorder>
  <zorder>equal_btn</zorder>
  <zorder>btn_7</zorder>
  <zorder>btn_8</zorder>
  <zorder>btn_9</zorder>
  <zorder>btn_5</zorder>
  <zorder>btn_6</zorder>
  <zorder>btn_3</zorder>
  <zorder>btn_2</zorder>
  <zorder>btn_1</zorder>
  <zorder>btn_0</zorder>
  <zorder>dot_btn</zorder>
  <zorder>btn_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
